src/frontends/pytorch/src/utils_quantize.cpp

```diff
@@ -0,0 +1,60 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "utils_quantize.hpp"
+
+#include "openvino/frontend/pytorch/node_context.hpp"
+
+namespace ov {
+namespace frontend {
+namespace pytorch {
+
+std::shared_ptr<QuantizedPtNode> quantize_per_tensor(std::shared_ptr<TorchDecoder> decoder,
+                                                     ov::Output<ov::Node> input,
+                                                     ov::Output<ov::Node> scale,
+                                                     ov::Output<ov::Node> zero_point,
+                                                     ov::Output<ov::Node> dtype) {
+    return std::make_shared<QuantizedPtNode>(QuantizedPtNodeType::QUANTIZE_PER_TENSOR,
+                                             decoder,
+                                             OutputVector{input, scale, zero_point, dtype},
+                                             decoder->num_of_outputs(),
+                                             false);
+}
+
+std::shared_ptr<QuantizedPtNode> dequantize(std::shared_ptr<TorchDecoder> decoder, ov::Output<ov::Node> input) {
+    return std::make_shared<QuantizedPtNode>(QuantizedPtNodeType::DEQUANTIZE,
+                                             decoder,
+                                             OutputVector{input},
+                                             decoder->num_of_outputs(),
```

Comment:
Not sure whether using `decoder->num_of_outputs()` is good idea to get number of outs for quantize/dequantize. This number of outputs will apply to number of outputs of node, for example quantized::add instead of real number of outs for quantize/dequantize. Would there be any cases where number of outputs for QuantizedPtNode is different than 1?